logger.go

```diff
@@ -238,9 +239,10 @@ func sliceStats(sample, histogramBins []float64) message.Fields {
 	if len(sample) == 0 {
 		return message.Fields{}
 	}
+	sort.Float64Slice(sample).Sort()
```

Comment:
Wouldn't it be simpler to use https://pkg.go.dev/sort#Float64s?

Reply:
totally!

Reply:
done